test/metabase/models/card_test.clj

```diff
@@ -775,3 +776,23 @@
                         Dashboard
                         before
                         changes)))))))))
+
+(deftest storing-metabase-version
+  (testing "Newly created Card should know a Metabase version used to create it"
+    (let [new-version "just a check"]
+      (t2.with-temp/with-temp [:model/Card card {}]
+        (is (= (:metabase_version card)
+               config/mb-version-string))
+
+        (create-card-revision! (:id card) false)
+
+        (let [rev (t2/select-one :model/Revision :model "Card" :model_id (:id card))]
+          (is (= (-> rev :object :metabase_version)
+                 config/mb-version-string)))
```

Comment:
nit: `rev` is only used once so we can remove the let binding

```suggestion
        (is (= (get-in (t2/select-one :model/Revision :model "Card" :model_id (:id card)) [:object :metabase_version])
               config/mb-version-string))
```

same with the test below

Reply:
This is a matter of style but I would disagree with you @qnkhuat here. The `rev` binding is naming something for readability.

Reply:
Scratch the above comment, it looks quite nice with a thread macro now